bundles/pom.xml

```diff
@@ -209,7 +210,7 @@
     <module>org.openhab.binding.systeminfo</module>
     <module>org.openhab.binding.tado</module>
     <module>org.openhab.binding.tankerkoenig</module>
-    <module>org.openhab.binding.telegram</module>    
+    <module>org.openhab.binding.telegram</module>
```

Comment:
Please remove this change.

Reply:
reverted - was changed because of auto-formatting the document on save :)